lib/preprocess.js

```diff
@@ -88,17 +87,24 @@ function replaceTextWithDummy(lintableTextArr, startIndex, length, dummy) {
 }
 
 function preprocessJs(text, filename) {
-  function dummyString(id) {
-    return `/* eslint-disable */{}/* ${HASH} ${id} *//* eslint-enable */`;
+  function wrapInEslintDisable(text) {
+    return `/* eslint-disable */${text}/* eslint-enable */`;
   }
-  return preprocess(text, filename, dummyString);
+
+  function toDummyString(id, matchText) {
+    if (matchText.startsWith("<%=")) {
+      return wrapInEslintDisable(`{}/* eslint-plugin-erb ${HASH} ${id} */`);
+    }
+    return wrapInEslintDisable(`/* eslint-plugin-erb ${HASH} ${id} */`);
+  }
```

Comment:
I'm just gonna leave this idea here, I don't feel like forking this repo today:
```suggestion
  function toDummyString(id, matchText) {
    let replacement = "";
    if (matchText.startsWith("<%=")) replacement = "{}";
    const tagged = `${replacement}/* eslint-plugin-erb ${HASH} ${id} */`;
    const wrapped = `/* eslint-disable */${tagged}/* eslint-enable */`;
    return wrapped;
  }
```